jetson/nucleo_bridge/test.cpp

```diff
@@ -78,7 +78,7 @@ float quadEnc(std::string name)
 {
     try
     {
-        ControllerMap::controllers[name]->quad_angle();
+        ControllerMap::controllers[name]->refresh_quad_angle();
```

Comment:
maybe it wouldn't be the worst thing to add an LCM unit test or smth

Reply:
added to TODO